Library/brew.sh

```diff
@@ -207,6 +192,9 @@ EOS
   esac
 fi
 
+source "$HOMEBREW_LIBRARY/Homebrew/utils/ruby.sh"
+setup-ruby-path
```

Comment:
Can this be moved to after the Bash command check is so it's only done for Ruby commands? Thanks.